frontend/src/pages/LineageView.tsx

```diff
@@ -32,6 +37,9 @@ import {
 import {
   MetadataStoreServicePromiseClient
 } from '../generated/src/apis/metadata/metadata_store_service_grpc_web_pb';
+import {getArtifactTypes} from '../components/LineageApi';
+import {getTypeName} from '../components/LineageUtils';
+import * as _ from 'lodash';
```

Comment:
If we're only using `groupby` maybe we can do:
[Importing a single function from lodash](https://stackoverflow.com/a/43479515/2162893)

Reply:
Done. Removed the full lodash dep in other parts of the package as well.